src/main/java/seedu/address/logic/commands/FindCommand.java

```diff
@@ -61,6 +56,41 @@ public boolean equals(Object other) {
         return predicate.equals(otherFindCommand.predicate);
     }
 
+    public static String getSpecificMessageUsage(String field) {
```

Comment:
Will it be necessary to handle invalid input fields here?

Reply:
I dont think that it would be necessary. the parser should handle parser exceptions for the message usage in my opinion